src/js/components/footer.js

```diff
@@ -3,37 +3,65 @@ const select = require("../utils/select");
 const { CLICK } = require("../events");
 const { prefix: PREFIX } = require("../config");
 
-const HIDDEN = "hidden";
 const SCOPE = `.${PREFIX}-footer--big`;
 const NAV = `${SCOPE} nav`;
 const BUTTON = `${NAV} .${PREFIX}-footer__primary-link`;
-const COLLAPSIBLE = `.${PREFIX}-footer__primary-content--collapsible`;
-
 const HIDE_MAX_WIDTH = 480;
 
+/**
+ * Expands selected footer menu panel, while collapsing others
+ */
 function showPanel() {
   if (window.innerWidth < HIDE_MAX_WIDTH) {
-    const collapseEl = this.closest(COLLAPSIBLE);
-    collapseEl.classList.toggle(HIDDEN);
-
-    // NB: this *should* always succeed because the button
-    // selector is scoped to ".{prefix}-footer-big nav"
-    const collapsibleEls = select(COLLAPSIBLE, collapseEl.closest(NAV));
+    const isOpen = this.getAttribute('aria-expanded') === 'true';
 
-    collapsibleEls.forEach((el) => {
-      if (el !== collapseEl) {
-        el.classList.add(HIDDEN);
-      }
+    // Close all other menus
+    document.querySelectorAll(BUTTON).forEach((button) => {
+      button.setAttribute('aria-expanded', false);
     });
+
+    this.setAttribute('aria-expanded', !isOpen);
   }
 }
 
+/**
+ * Swaps the <h4> element for a <button> element (and vice-versa)
+ *
+ * @param {Boolean} isMobile - If the footer is in mobile configuration
+ */
+function toggleHtmlTag(isMobile) {
+  const footer = document.querySelector(SCOPE);
+  const primaryLinks = footer?.querySelectorAll(BUTTON);
+  const newElementType = isMobile ? 'button' : 'h4';
+
+  primaryLinks?.forEach(currentElement => {
+    const currentElementClasses = currentElement.getAttribute('class');
+
+    // Create the new element
+    const newElement = document.createElement(newElementType);
+    newElement.setAttribute('class', currentElementClasses);
+    newElement.classList.toggle(`${PREFIX}-footer__primary-link--button`, isMobile);
+    newElement.textContent = currentElement.textContent;
```

Comment:
If the new element is button outside of a form, we should add `type="button"` from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button#attr-type).

Reply:
✅